packages/checkout-extension/src/ExtensionService.ts

```diff
@@ -0,0 +1,111 @@
+import {
+    CheckoutService,
+    Extension,
+    ExtensionCommandType,
+    ExtensionRegion,
+} from '@bigcommerce/checkout-sdk';
+
+export class ExtensionService {
+    private _handlerRemovers: { [extensionId: string]: Array<() => void> } = {};
+
+    constructor(
+        private _checkoutService: CheckoutService,
+        private _setIsShowLoadingIndicator: (show: boolean) => void,
+    ) {}
+
+    async loadExtensions(): Promise<void> {
+        await this._checkoutService.loadExtensions();
+    }
+
+    async renderExtension(container: string, region: ExtensionRegion): Promise<void> {
+        const extension = this._getExtensionByRegion(region);
+
+        if (!extension) {
+            return;
+        }
+
+        await this._checkoutService.renderExtension(container, region);
+
+        if (!this._handlerRemovers[extension.id]) {
+            this._handlerRemovers[extension.id] = [];
+        }
+
+        this._handlerRemovers[extension.id].push(
+            this._checkoutService.handleExtensionCommand(
+                extension.id,
+                ExtensionCommandType.ReloadCheckout,
+                () => {
+                    void this._checkoutService.loadCheckout(
```

Comment:
Does this `void` do anything, considering you're using `() => {}`?

Reply:
Yes, eslint suggests marking the returned promise from `this._checkoutService.loadCheckout` as ignored with a `void` operator. Otherwise, I need to change it to `async () => void`.